images/source/origin-local-release.repo

```diff
@@ -0,0 +1,4 @@
+[origin-local-release]
+name = OpenShift Origin Release from Local Source
+baseurl = file:///srv/origin-local-release/
```

Comment:
@smarterclayton I am mounting this in during the `imagebuilder` build -- how do you want to fall back if it is not present? I can just build a new image on top of this that actually adds in the RPMs and over-write the previous `openshift/origin-source`. Does that sound reasonable? Someone building without `imagebuilder` would pay the space cost but that's about it.